### DISCUSSION TITLE
[Eng] DevExp Wishlist

### DISCUSSION THREAD

**Discussion by 0xemc** [KlimaDAO/klimadao#1240] [Ideas]
### DevExp Wishlist
| Name | Benefit | Link | Difficulty | Priority | Issue |
|--------|--------|--------|--------|--------|-------|
| Doppler | Isolated config, separated from Vercel | https://www.doppler.com/ | Easy | Low | - |
| Shared Lint and TSconfig package | Shared single tsconfig and lint config across the monorepo | - | Medium | Medium |- |
| Blocking Github actions | test and linting executed before vercel deploys, stops breaking PRs | - | Easy | High | https://github.com/KlimaDAO/klimadao/issues/1245 |
| 2 PR Branch protection | Require 2 reviews (or manual override) for PRs to staging | - | Easy | High | https://github.com/KlimaDAO/klimadao/issues/1246 |
| End to End Typegen | Remove risks and effort involved with manually maintained types |  https://the-guild.dev/graphql/codegen | Easy | High | https://github.com/KlimaDAO/klimadao/issues/1247 |
| Eng github team | Isolate permissions and easier tagging |  https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team | Easy | Low | - |
| Merge queues | Isolate permissions and easier tagging |  https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team | Easy | Low | - |
| Ticket and Milestone Templates |Repeatable task and bug templates |https://gist.github.com/0xemc/a4f2723a8aaee0157125e276b57955cb  | Easy | Low | - |

**Comment by 0xemc** [comment]
@Atmosfearful @ladytrekker @0xMakka @jabby09 @Najada  feel free to contribute/comment

**Comment by jabby09** [comment]
@0xemc created an epic here https://github.com/KlimaDAO/klimadao/milestone/18

For easy difficulty and high priority, created tickets; assigned to you and added link to the table above.

**Comment by 0xemc** [comment]
Moving to

https://www.notion.so/klima-dao/Engineering-5e31593c6b124f1eb25ff6813c0f64be?pvs=4#8a4d1decbf8649c78801e592d1c46bd4
